src/main/java/Exercise.java

```diff
@@ -7,10 +7,10 @@
 
 public class Exercise {
 
-    private ArrayList<String> type;
-    private ArrayList<String> minorMuscleExercised;
-    private ArrayList<String> majorMuscleExercised;
-    private ArrayList<String> equipmentNeeded;
+    private final ArrayList<String> type;
```

Comment:
I assume that the variables in this class will be directly from the database. If this is the case then it's good for merging.